app/controllers/admin/departments_controller.rb

```diff
@@ -17,7 +17,7 @@ class DepartmentsController < Admin::ApplicationController
     # for more information
 
     def find_resource(param)
-      Department.find_by! slug: param
+      Department.find param
```

Comment:
It's a really good point - id vs. slug has an impact on navigation by URL, and makes things less readable. Is there a practical reason?